src/app/modules/group/components/group-invite-users/group-invite-users.component.ts

```diff
@@ -23,15 +23,16 @@ export class GroupInviteUsersComponent {
   @Input() group?: Group
   @Output() refreshRequired = new EventEmitter<void>();
 
-  @ViewChild(TextareaComponent) private textArea?: TextareaComponent;
-
   state: 'empty'|'too_many'|'loading'|'ready' = 'empty';
+  inviteForm = this.formBuilder.group({ invitations: '' });
+  inputName = 'invitations'
```

Comment:
Mark it as a constant
```suggestion
  readonly inputName = 'invitations'
```